spec/controllers/odk_form_download_spec.rb

```diff
@@ -18,54 +18,58 @@
       @form3.publish!
     end
 
-    describe 'listing forms' do
-      it 'should succeed' do
-        get_s(odk_form_list_path)
+    describe "listing forms" do
+      it "should succeed" do
+        get_s(odk_form_list_path, format: :xml)
```

Comment:
Should we perhaps hard-code the paths in this spec instead of using the helpers? At first glance I assumed this would construct a path that ends in `.xml`, which nobody actually uses. This may not be true but I can't be sure and it seems that we should be testing the real paths publish in our documentation instead of what the helper generates?